docker/nuxeo-explorer-docker/skaffold.yaml

```diff
@@ -18,7 +18,7 @@ build:
         buildContext:
           localDir: {}
         buildArgs:
-          BASE_IMAGE: ${DOCKER_REGISTRY}/${ORG}/nuxeo:11.x
+          BASE_IMAGE: ${DOCKER_REGISTRY}/${ORG}/nuxeo:${NUXEO_IMAGE_VERSION}
```

Comment:
level=fatal msg="creating runner: parsing skaffold config: parsing api version: yaml: line 21: mapping values are not allowed in this context" (?)

Reply:
fixed by https://github.com/nuxeo/nuxeo-explorer/commit/5d44906f9668372d5f9e64ce10fafa63a7ef2ab2